File: src/test/java/com/example/stopgap/uuid/UuidGenTest.java
package com.example.stopgap.uuid;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

final class UuidGenTest {

    @Test
    void differentUuids() {
        final var uuidGen = new UuidGen();
        final var uuid1 = uuidGen.generate();
        final var uuid2 = uuidGen.generate();
        assertThat(uuid1).isNotEqualTo(uuid2);
    }

}

File: src/test/kotlin/com/example/stopgap/generator/uuid/UuidGenTest.kt
package com.example.stopgap.generator.uuid

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class UuidGenTest {
    @Test
    fun differentUuids() {
        val uuidGen = UuidGen()
        val uuid1 = uuidGen.generate()
        val uuid2 = uuidGen.generate()
        assertThat(uuid1).isNotEqualTo(uuid2)
    }
}